analytics/aggregate.go

```diff
@@ -836,3 +836,12 @@ func AggregateData(data []interface{}, trackAllPaths bool, ignoreTagPrefixList [
 
 	return analyticsPerOrg
 }
+
+func TrimTag(thisTag string) string {
+	trimmedTag := strings.TrimSpace(thisTag)
+
+	if strings.Contains(trimmedTag, ".") {
+		trimmedTag = strings.Trim(trimmedTag, ".")
```

Comment:
@sredxny will `strings.Trim` remove all the `.` from the tag?

Reply:
@tbuchaillot now it will. I did the change. I just noted that for the blank spaces it will remove only the leading and trailing spaces removed. If we have a tag with and empty space in the middle it will not work and the blank space will remain, is it expected?